gensim/sklearn_api/basemodel.py

```diff
@@ -11,7 +11,7 @@
 from abc import ABCMeta, abstractmethod
 
 
-class BaseSklearnWrapper(object):
+class BaseTransformer(object):
```

Comment:
hi,

jumping in the conversation a little late here, but I do agree that the old `BaseSklearnWrapper` does very little since

 - the default `get_params` and `set_params` from sklearn's `BaseEstimator` are very reasonable and should only be overriden when necessary; if all the concrete wrapper classes inherit from `BaseEstimator` (as they do now) then there's no need to force them to re-implement these methods
 - @tmylk `partial_fit` is absolutely not a requirement for a class to be compatible with sklearn `Pipeline` since lots of algorithm are not online in nature. If there is a particular algorithm that most people are confused about, then the warning should be raised for that class only.